Library/Homebrew/formula_installer.rb

```diff
@@ -202,7 +202,10 @@ def install
 
     oh1 "Installing #{Tty.green}#{formula.full_name}#{Tty.reset}" if show_header?
 
-    report_analytics_event("install", formula.full_name)
+    unless exception.formula.tap.private?
```

Comment:
Looks like this should have been just `formula.tap.private?` for the tests not to fail.